Framework/Kernel/inc/MantidKernel/case_insensitive_map.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CASE_INSENSITIVE_MAP_H
```

Comment:
Is there a reason that the filename doesn't follow the standard conventions without underscores?